web/crux/package.json

```diff
@@ -21,6 +21,9 @@
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
+    "encrypt:generate": "node dist/main encrypt generate",
+    "encrypt:migrate": "node dist/main encrypt migrate",
+    "encrypt:rotate": "node dist/main encrypt rotate",
```

Comment:
Based on these commands encryption only works if we build the project?

Reply:
Yep. Haven't checked yet if we can pass args to `nest start`.

Reply:
Okey, no probs.